src/test/scala/wdl4s/expression/ValueEvaluatorSpec.scala

```diff
@@ -512,4 +512,16 @@ class ValueEvaluatorSpec extends FlatSpec with Matchers {
     val expectedMessage = "Sorry! Operation + is not supported on empty optional values. You might resolve this using select_first([optional, default]) to guarantee that you have a filled value."
     hello.add(noneWorld) should be(Failure(OptionalNotSuppliedException("+")))
   }
+
+  "Ternary if blocks" should "evaluate LHS if the condition is true" in {
+    constEval(""" if (5 == 4 + 1) then 6 + 7 else 14 * 15 """) should be(WdlInteger(13))
```

Comment:
since I was @ summoned...

would it be appropriate to check that only one of these expressions is actually **evaluated** (rather than only checking that the return value is correct)